server/events/markdown_renderer.go

```diff
@@ -258,7 +263,7 @@ func (m *MarkdownRenderer) renderProjectResults(results []command.ProjectResult,
 	default:
 		return "no template matched–this is a bug"
 	}
-	return m.renderTemplate(tmpl, resultData{resultsTmplData, common})
+	return m.renderTemplateTrimSpace(tmpl, resultData{resultsTmplData, common})
```

Comment:
I added 2 kinds of TrimSpace.

- terraform logs TrimSpace like `resultData.Rendered = m.renderTemplateTrimSpace(templates.Lookup("importSuccessUnwrapped"), result.ImportSuccess)`
- full results TrimSpace (here)